base/operators.jl

```diff
@@ -1060,7 +1086,8 @@ julia> filter(!isletter, str)
 "∀  > 0, ∃  > 0: |-| <  ⇒ |()-()| < "
```

Comment:
The docs for `!f` should be updated to say that it returns a `ComposedFunction`, with a compat note indicating that this is new in 1.9